stdlib/3/unicodedata.pyi

```diff
@@ -1,37 +1,36 @@
 # Stubs for unicodedata (Python 3.4)
-#
-# NOTE: This dynamically typed stub was automatically generated by stubgen.
+from typing import Any, TypeVar, Union
 
-from typing import Any
-
-ucd_3_2_0 = ...  # type: Any
+ucd_3_2_0 = ...  # type: UCD
 ucnhash_CAPI = ...  # type: Any
 unidata_version = ...  # type: str
 
-def bidirectional(unichr): ...
-def category(unichr): ...
-def combining(unichr): ...
-def decimal(chr, default=...): ...
-def decomposition(unichr): ...
-def digit(chr, default=...): ...
-def east_asian_width(unichr): ...
-def lookup(name): ...
-def mirrored(unichr): ...
-def name(chr, default=...): ...
-def normalize(form, unistr): ...
-def numeric(chr, default=...): ...
+_default = TypeVar('_default')
+
+def bidirectional(unichr: str) -> str: ...
+def category(unichr: str) -> str: ...
+def combining(unichr: str) -> int: ...
+def decimal(chr: str, __default: _default=...) -> Union[int, _default]: ...
```

Comment:
TBH I'd leave the `Union` approach as is because

1. it's the most correct one (we don't ignore the type)
2. it's consistent with the Python 2 `unicodedata` module